Beqa Gogberashvili/Challenge2/index.php

```diff
@@ -0,0 +1,177 @@
+        } else {
+
+            curl_close($handle);
+            $opts = [
+                'http' => [
+                    'ignore_errors' => true,
+                    'method' => 'GET',
+                    'header' => [
+                        'User-Agent: PHP'
+                    ]
+                ]
+            ];
+
+            // Getting repos and followers quantity number:
+
+            $ch = curl_init();
+            curl_setopt_array($ch, [
+                CURLOPT_URL => "https://api.github.com/users/" . $gitUser,
+                CURLOPT_HTTPHEADER => [
+                    "Accept: application/vnd.github.v3+json",
+                    'User-Agent: PHP'
+                ],
+                CURLOPT_RETURNTRANSFER => true
+            ]);
+            $response = curl_exec($ch);
+            curl_close($ch);
+            $arr = json_decode($response);
+            $followers = $arr->followers;
+            $public_repos = $arr->public_repos;
+
+            $timesF = ceil($followers / 21);
+            $timesR = ceil($public_repos / 100);
+
+            // Generating user data
+
+            $context = stream_context_create($opts);
+
+            echo '<form method="post">
+            <a href="./index.php"><input type="submit" name="back" class="back" value=" Go back"></a>
+        </form>';
+            if (isset($_POST['choice']) && $_POST['choice'] == 'repos') {
+                $choice = 'repos';
+            }
+            if (isset($_POST['choice']) && $_POST['choice'] == 'followers') {
+                $choice = 'followers';
+            }
+            if (isset($_POST['choice']) && $_POST['choice'] == 'both') {
+                $choice = 'both';
+            }
+            if (isset($_GET['choice']) && $_GET['choice'] == 'repos') {
+                $choice = 'repos';
+            }
+            if (isset($_GET['choice']) && $_GET['choice'] == 'followers') {
+                $choice = 'followers';
+            }
+            if (isset($_GET['choice']) && $_GET['choice'] == 'both') {
+                $choice = 'both';
+            }
+
+
+            if ((isset($_POST['choice']) && $_POST['choice'] == 'repos') || (isset($_POST['choice']) && $_POST['choice'] == 'both') || (isset($_GET['choice']) && $_GET['choice'] == 'repos') || (isset($_GET['choice']) && $_GET['choice'] == 'both')) {
+                echo '<div class="repos">            
+            <h1>Repos of <a href="https://github.com/' . $gitUser . '"><span style="border-bottom: 1px solid #dedede; color: white !important">' . $gitUser . '</a></span> :</h1>
+            <ul>';
+
+                // Repos generator
+
+                $num = 1;
+                for ($e = 0; $e < $timesR; $e++) {
+                    ${'reposContent' . $e} = file_get_contents("https://api.github.com/users/" . $gitUser . "/repos?per_page=100&page=" . $e + 1, false, $context);
+                    ${'reposArr' . $e} = json_decode(${'reposContent' . $e});

```

Comment:
აქ უნდა ამიხსნა რას შვები.

Reply:
იმდენჯერ ვატრიალებ for-ს რამდენი ასეული რეპოც აქვს იუზერს. როცა 100-ზე მეტი აქვს, მაინც ყველა შედეგს გამოიტანს